.github/workflows/ci.yaml

```diff
@@ -59,3 +59,6 @@ jobs:
 
   nox:
     uses: ./.github/workflows/reusable-nox.yml
+
+  spell-check:
```

Comment:
You can add this to the matrix in reusable-nox.yml instad of creating a whole new file:

https://github.com/ansible/ansible-documentation/blob/18a7692cff6ac5f8e8bd65fc1e70a8504d85892e/.github/workflows/reusable-nox.yml#L12-L19

Reply:
good idea. I wasn't really sure if should go in there tbh.